app/src/components/Loader.js

```diff
@@ -221,7 +221,7 @@ export const DataLoader = () => {
       Get medicalFiles
       */
     if (response.data.medicalFiles || initialLoad) {
-      setLoading('Chargement des informations médicales');
+      setLoading('Chargement des informations de santé');
```

Comment:
```suggestion
      setLoading('Chargement des dossiers médicaux');
```

en l'occurence, `medicalFile` n'est pas l'information de santé :)